### DISCUSSION TITLE
Question about type annotations

### DISCUSSION THREAD

**Discussion by shomodj** [kdl-org/kdl#398] [General]
I'm trying to signal to parser a type annotation that is a list of strings or map of string:string, but this is not allowed as it's constrained by the identifier

```
(list[string])    # can't have it
(list<string>) # can't have it

# works but not my favourite
(list:string)
(map:string:string) 
```

I was wondering if this can discussed?
Thanks

**Comment by tabatkins** [comment]
A type annotation is just a string; it's *usually* written as an ident string `(foo)`, but you can use the quoted forms as well `("foo")`.

So if you want to smuggle in values that aren't valid idents, just quote them, like `("list[string]")node`

**Reply by shomodj** [reply]
Ahh, I missed that, thanks
